sharded_queue/api.lua

```diff
@@ -40,11 +49,19 @@ function sharded_tube.put(self, data, options)
     options.tube_name = self.tube_name
     options.bucket_id = bucket_id
     options.bucket_count = bucket_count
+    options.options = {
+        log_request = utils.normalize.log_request(options.log_request)
```

Comment:
Seems that normalize is useless here (we do not validate other options)

Reply:
I think, there is a substitution of concepts: in fact, the parameters above are not optional, but mandatory, their invalidity leads to an error. And the `log_request` is an optional parameter, and it's invalidity does not lead to errors, however, we should ignore the user's attempts to pass a string or number 42 into this parameter.
Therefore, I think `normalize` is necessary.